code/game/gamemodes/abduction/abduction.dm

```diff
@@ -24,7 +25,8 @@
 	world << "<b>Crew</b> - don't get abducted and stop the abductors."
 
 /datum/game_mode/abduction/pre_setup()
-	abductor_teams = max(1, min(max_teams,round(num_players()/config.abductor_scaling_coeff)))
+	// abductor_teams = max(1, min(max_teams,round(num_players()/config.abductor_scaling_coeff))) Old scaling
+	abductor_teams = max(1, min(max_teams,round(max_teams*num_players()/max_players))) // Scaling based off of max players
```

Comment:
Well we could put max teams and max players into the config file, and then you can set those vars by reading from the file?